src/components/notes/note-input-form/index.jsx

```diff
@@ -4,16 +4,18 @@ import { RichTextEditor } from "../text-editor";
 import { ColorPalette } from "../color-palette";
 import { PriorityList } from "../priority-list";
 import { LabelEditor } from "../../label-editor";
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import { postNoteService } from "../../../services/notes-services/post-note-service";
 import { useAuth } from "../../../contexts/auth-context";
```

Comment:
Suggestion: create an index page and import all contexts here. and through this you can import any contextx in a single line like
import { useAuth, useNote } from "../../../contexts";